lib/plugins/aws/package/lib/generateCoreTemplate.test.js

```diff
@@ -6,9 +6,9 @@ const expect = require('chai').expect;
 const AwsProvider = require('../../provider/awsProvider');
 const Serverless = require('../../../../Serverless');
 const validate = require('../../lib/validate');
-const configureStack = require('../lib/configureStack');
+const generateCoreTemplate = require('./generateCoreTemplate');
 
-describe('#configureStack', () => {
+describe('#generateCoreTemplate', () => {
```

Comment:
Should have a `()` at the end 😬 --> `#generateCoreTemplate()`.

(and all other `describe` blocks in this file as well)